asapdiscovery-data/asapdiscovery/data/schema/ligand.py

```diff
@@ -115,7 +115,15 @@ class Ligand(DataModelAbstractBase):
         description="Expansion tag linking this ligand to its parent in a state expansion if needed",
     )
 
-    tags: dict[str, str] = Field({}, description="Dictionary of SD tags")
+    tags: dict[str, str] = Field(
+        {},
+        description="Dictionary of SD tags. "
+        "If multiple conformers are present, these tags represent the first conformer.",
+    )
+
+    conf_tags: Optional[dict[str, list]] = Field(
+        {}, description="Dictionary of SD tags for each conformer."
+    )
```

Comment:
There should be only one API point for getting tags IMO, otherwise error prone.

Reply:
Unless this is a technical impossibility, or there are serious reasons not to.

Reply:
We can discuss. :)

Reply:
This is worth a discussion. This was a choice to avoid breaking backward compatibility.

Reply:
I think the cleanest way to do this going forward would be to replace `tags` with what `conf_tags` is.